api/notd/messages.py

```diff
@@ -37,3 +38,11 @@ class UpdateCollectionTokensMessageContent(MessageContent):
     _COMMAND = 'UPDATE_COLLECTION_TOKENS'
     address: str
     shouldForce: Optional[bool]
+
+class UpdateActivityForAllCollectionsMessageContent(MessageContent):
+    _COMMAND = 'UPDATE_ACTIVITY_FOR_ALL_COLLECTIONS'
+
+class UpdateActivityForCollectionMessageContent(MessageContent):
+    _COMMAND = 'UPDATE_COLLECTION_ACTIVITY'
+    registryAddress: str
```

Comment:
we should be consistent with varaible names here. generally when we deal with collections we just use address i think. please confirm and then update if its the case